molior/ops/aptly.py

```diff
@@ -20,6 +21,9 @@
 
 
 def debchanges_get_files(sourcepath, sourcename, version, arch="source"):
+    # strip epoch versions
+    m = re.match("(\d+:)?(\d.+)", version)
```

Comment:
Improve by making the first parenthesis a no capturing group using `?:`

Reply:
done, thanks!